### ISSUE TITLE
Priority auto-complete adds unnecessary character when editing priority

### ISSUE THREAD

**Issue by cjrobe** [mNantern/QTodoTxt#129]
If I have this entry in my tasks:
(A) +todo fix my muffler

And I edit it by replacing the A with a B, the priority auto-complete pops up. If I ignore it and click away, it's OK, but if I accept the auto-complete by clicking it or pressing enter, it adds an extra ending parenthesis like so:

(B) ) +todo fix my muffler

**Comment by mimmori** [comment]
yes, very annoying behaviour. For now i use a workaround: change priority via +/- key.

**Comment by cjrobe** [comment]
That is a very helpful tip mimmori, thanks.
